docs/rollup.config.js

```diff
@@ -0,0 +1,133 @@
+import resolve from '@rollup/plugin-node-resolve';
```

Comment:
Why do the docs have a separate rollup config file? The demo was added to the same config as the core library. Since the docs are meant to replace this demo, either it follows the same structure or we move the library's config to its folder (attractions)

Reply:
A separate config file is handier somehow. As for the demo, yeah, maybe we should move the demo's config into the demo folder, and the core library doesn't even use Rollup, right?

Reply:
The core library does use Rollup for building before publishing to npm (to have the non-Svelte versions).
I thought we will remove the demo entirely once the docs are ready?

Reply:
Yes, but that time hasn't come yet. The source code of the demo is still useful